packages/core/src/amazonq/lsp/lspController.ts

```diff
@@ -0,0 +1,378 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import * as path from 'path'
+import * as fs from 'fs-extra'
```

Comment:
do not use fs-extra. we have a `fs.ts` module that has most of its features.

Reply:
1. I need to use `const file = fs.createWriteStream(localFile)`. This is not supported in current fs.ts
2. `fs-extra` is widely used across this code base.
3. This feature does not support Web

Reply:
> need to use const file = fs.createWriteStream(localFile)

ok,  fs.ts doesn't have an answer for that yet.

> fs-extra is widely used across this code base

we are in the process of removing it. using it more adds work :)

> This feature does not support Web

that might be the case now, but it can change in the future. and web support in general is more feasible when most modules are using fs.ts instead of node-only things.